<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>680</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>680</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../src.qrc">:/images/Home.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>搜索书籍</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QComboBox" name="comboBox">
               <property name="statusTip">
                <string>Search by</string>
               </property>
               <item>
                <property name="text">
                 <string>ISBN</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>所有书籍</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>标题</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>作者</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>出版社</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>年份</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_search_book">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../src.qrc">
                 <normaloff>:/images/search-icon.png</normaloff>:/images/search-icon.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>21</width>
                 <height>21</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="tableView"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../src.qrc"/>
 </resources>
 <connections/>
</ui>
